app/presenters/tree_node/custom_button.rb

```diff
@@ -1,7 +1,7 @@
 module TreeNode
   class CustomButton < Node
-    set_attribute(:image) do
-      @object.options && @object.options[:button_image] ? "100/custom-#{@object.options[:button_image]}.png" : '100/leaf.gif'
+    set_attribute(:icon) do
+      @object.options && @object.options[:button_image] ? "product product-custom-#{@object.options[:button_image]}" : 'fa fa-file-o'
```

Comment:
@skateman I'm not sure what you mean. We have fonticon equivalents of the 15 custom icons.